docs/core/components/about-github-link.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: About Github Link Component
+Added: v1.0.0
+Status: Experimental
+Last reviewed: 2019-09-09
+---
+
+# [About Github Link Component](../../../lib/core/about/about-github-link/about-github-link.component.ts "Defined in about-github-link.component.ts")
+
+Shows which version of the application you are running based on the latest commit.
+Shows the server settings of the application that you are running.
+
+## Basic Usage
+
+With default input values
+
+```html
+<adf-about-github-link></adf-about-github-link>
+```
+
+With custom input values:
+
+```html
+<adf-about-github-link
+    githubUrlCommitAlpha="YourUrl"
```

Comment:
the component tag already says "github-link", so we can remove that from the property name. Also I would suggest just calling the property "url" as we may want to display something else than "alpha" in the future. What do you think @eromano ?

Reply:
good idea